control/runner.go

```diff
@@ -256,10 +256,10 @@ func (r *runner) HandleGomitEvent(e gomit.Event) {
 				pool.IncRestartCount()
 
 				runnerLog.WithFields(log.Fields{
-					"_block":        "handle-events",
-					"event":         v.Name,
-					"aplugin":       v.Version,
-					"restart_count": pool.RestartCount(),
+					"_block":         "handle-events",
+					"plugin-name":    v.Name,
+					"plugin-version": v.Version,
```

Comment:
I'm not clear what the difference is between v.Name + v.Version and v.String. Right before we log this we log an error with "aplugin": v.String. Not sure what the difference is but it seems like we should be consistent? I tried looking into core/control_event for the definitions of DeadAvailablePluginEvent but it's not commented on what the fields are meant to hold.